index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portofolio</title>
+</head>
+<body>
+  <div class="container">
+    <div class="toolbar">
+      <div class="navbar">
+        <span class="navbar_welcome">Welcome</span>
+        <div class="navbar_menu">
+          <div class="navbar_menu_button">
+            <img src="assets/menu_icon.svg" alt="menu_icon">
+          </div>
+        </div>
+        
+      </div>
+      
+    </div>
+    <div class="main_section">
+      <h1 class="main_section_h1">Hey there. I'm Koami.</h1>
+      <h2 class="main_section_h2">I'm a software developer</h2>
+      <p class="main_section_p">
+        professionally trained by microverse online university , 
+        I can help in custom software development (according to demand), 
+        website development, mobile application development,
+         network maintenance and many other IT areas.
+      </p>
```

Comment:
- [ ] Kindly add the appropriate margin or padding to the text description to match the Figma design template you chose to make your UI look good 👍🏼. Currently, your text is touching the edges of the container.

Your Design 👇🏼 

![image](https://user-images.githubusercontent.com/63473584/185478498-a0ede904-5017-4b3a-b975-2c326f23345a.png)

Figma Design 👇🏼 
![image](https://user-images.githubusercontent.com/63473584/185478749-7d3874b6-ed55-4640-a40a-d8ddc9b166d6.png)